### PR TITLE
Various policies

### PR DESCRIPTION
**Pull request by vegazbabz**
3 different policies.
- 1 for auditing if auto shutdown has been configured for VMs (not just 1 VM as in ALZ)
- 1 deny the changes of retention days settings in the LAW to lower than a minimum
- 1 DINE for audit_log_enabled = ON on MySQL DB

### CODE REVIEW

File: policyDefinitions/Monitoring/deny-change-of-retention-days-in-log-analytics-workspace/azurepolicy.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{
+  "name": "c7eedeb9-bf27-40fa-91ce-4ccb1ae35edc",
+  "type": "Microsoft.Authorization/policyDefinitions",
+  "properties": {
+    "displayName": "Deny change of retention days in Log Analytics workspace",
```
Comment (aschabus) [inline policyDefinitions/Monitoring/deny-change-of-retention-days-in-log-analytics-workspace/azurepolicy.json:5]: Please change the display name as it's confusing that it says "Deny...", but the policy can be configured to other effects too.

File: policyDefinitions/SQL/ensure-server-parameter-'audit_log_enabled'-is-on-for-mysql-db-server/azurepolicy.json [hunk 2]
```diff
@@ -0,0 +1,110 @@
+{
+  "name": "4dc90661-5d91-41f1-be00-d243f6f3fe9c",
+  "type": "Microsoft.Authorization/policyDefinitions",
+  "properties": {
+    "displayName": "Deploy - Ensure server parameter 'audit_log_enabled' is set to 'ON' for MySQL Database Server",
```
Comment (aschabus) [inline policyDefinitions/SQL/ensure-server-parameter-'audit_log_enabled'-is-on-for-mysql-db-server/azurepolicy.json:5]: Please change the display name as it's confusing that it's says "Deploy...." but allows other effects to.
e.g. Ensure audit log is enabled for MySQL.

**Review by aschabus — CHANGES_REQUESTED** [review]
Please look through the comments.

**Comment by vegazbabz** [comment]
> Please look through the comments.

@aschabus I addressed your comments now.

**Pull request merged**